src/System.CommandLine/Command.cs

```diff
@@ -87,18 +148,37 @@ private protected override void AddSymbol(Symbol symbol)
 
         internal List<ValidateSymbol<CommandResult>> Validators { get; } = new List<ValidateSymbol<CommandResult>>();
 
+        /// <summary>
+        /// Adds a custom <see cref="ValidateSymbol{T}(CommandResult)"/> to the command. Validators can be used
+        /// to create custom validation logic.
+        /// </summary>
+        /// <param name="validate">The delegate to validate the symbols during parsing.</param>
         public void AddValidator(ValidateSymbol<CommandResult> validate) => Validators.Add(validate);
 
+        /// <summary>
+        /// Gets or sets a value that indicates whether unmatched tokens should be treated as errors. For example,
+        /// if set to <c>true</c> and an extra command or argument is provided, validation will fail.
+        /// </summary>
         public bool TreatUnmatchedTokensAsErrors { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets the <see cref="ICommandHandler"/> for the command. The handler represents the action
+        /// that will be performed when the command is invoked.
+        /// </summary>
```

Comment:
It might be worth adding a remark that command line input specifying commands that don't have handlers will result in parse errors.

Reply:
OK I also added a reference to the `CommandHandler.Create` method. For me, this was one of the trickiest things "discover". We may want to expand the documentation even more. Take a look.